lib/tasks/add_stockit_packages_pieces_weight.rake

```diff
@@ -0,0 +1,23 @@
+namespace :stockit do
+  task add_stockit_packages_pieces_weight: :environment do
+    offset = 0
+    per_page = 1000
+    loop do
+      items_json = Stockit::ItemSync.new(nil, offset, per_page).index
+      offset += per_page
+      stockit_items = JSON.parse(items_json["items"])
+      bar = RakeProgressbar.new(stockit_items.size)
+      next if stockit_items.blank?
+
+      stockit_items.each do |value|
+        bar.inc
+        next if value["id"].blank?
+
+        package = Package.find_by(stockit_id: value["id"])
+        package&.update_attributes(pieces: value["pieces"], weight: value["weight"])
```

Comment:
If am not wrong, this will trigger live updates of package updates. Do we want to have sync updates? or need to skip here?
@abhinav098 @namrataukirde

Reply:
True @swatijadhav.

@abhinav098 It will cause a loop and we don't want that to happen. Have you tested this rake on local? 

I will suggest to test it on local with proper set of data and check the documentation of update_attributes.

Thanks.

Reply:
Yes @namrataukirde @swatijadhav , I tested this rake on local, It didn't go into any loop. To to avoid any kind of confusion, i'll use `update_column` here.

Reply:
Used `update_columns` now for updating the package record. It will skip the validations and callbacks. 
https://apidock.com/rails/ActiveRecord/Persistence/update_columns